p4c.lsm

```diff
@@ -0,0 +1,18 @@
+Begin4
+Title:          p4c
+Version:        1.2.0+g202103251118~269c3d
+Entered-date:   2021-03-25
+Description:    p4c is a new, alpha-quality reference compiler for the P4
+                programming language. It supports both P4-14 and P4-16; you
+                can find more information about P4 here and the specifications
+                for both versions of the language in http://p4.org.
+                p4c is modular; it provides a standard frontend and midend
+                which can be combined with a target-specific backend to create
+                a complete P4 compiler. The goal is to make adding new
+                backends easy.
+Keywords:       P4, Compiler
```

Comment:
Same comment about this description here being obsolete, as for earlier similar comment.

Reply:
Fixed.